django/contrib/admin/tests.py

```diff
@@ -26,26 +55,38 @@ class AdminSeleniumWebDriverTestCase(StaticLiveServerTestCase):
         'django.contrib.sessions',
         'django.contrib.sites',
     ]
-    webdriver_class = 'selenium.webdriver.firefox.webdriver.WebDriver'
 
-    @classmethod
-    def setUpClass(cls):
-        if not os.environ.get('DJANGO_SELENIUM_TESTS', False):
+    def _get_local_webdriver_class(self, spec):
+        """
+        Returns the webdriver for a local browser corresponding to the given
+        browser spec.
+        """
+        browsers = {
```

Comment:
Can you check if there's a way to pull this mapping from selenium or build it automatically so that if browsers are added there, we don't have to update Django?